samples/matter/template/boards/nrf54h20_suit_overlay.conf

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable SUIT services
+CONFIG_SUIT=y
+CONFIG_MGMT_SUITFU=y
+CONFIG_MGMT_SUITFU_GRP_SUIT=y
+CONFIG_SSF_SUIT_SERVICE_ENABLED=y
+
+# Enable mcumgr for SUIT purposes
+CONFIG_ZCBOR=y
+CONFIG_ZCBOR_CANONICAL=y
+CONFIG_MCUMGR=y
+CONFIG_BOOTLOADER_MCUBOOT=n
+CONFIG_IMG_MANAGER=n
+CONFIG_MCUMGR_SMP_LEGACY_RC_BEHAVIOUR=y
+
+# Allow for large Bluetooth data packets.
+CONFIG_BT_L2CAP_TX_MTU=498
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+CONFIG_BT_BUF_ACL_TX_SIZE=502
+
+# Enable the Bluetooth mcumgr transport (unauthenticated).
+CONFIG_MCUMGR_TRANSPORT_BT=y
+CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN=n
+CONFIG_MCUMGR_TRANSPORT_BT_CONN_PARAM_CONTROL=y
+CONFIG_MCUMGR_TRANSPORT_BT_REASSEMBLY=y
+CONFIG_MCUMGR_TRANSPORT_NETBUF_SIZE=2475
```

Comment:
Why 2475? We have this buffer set to 1024 for other configurations and we use the same MTU size, so either this one does not have to be that big, or we should increase it for other configurations as well.

Reply:
I will investigate it. This value was proposed by the SUIT team. Maybe the lower value indeed is better.

Reply:
They seem redundant to us. I removed them.